pkgs/tools/security/osv-scanner/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  testers,
+  osv-scanner,
+}:
+buildGoModule rec {
+  pname = "osv-scanner";
+  version = "1.0.1";
```

upstream has new release
```suggestion
  version = "1.0.2";
```